src/analysis/taint_analysis.rs

```diff
@@ -137,41 +133,45 @@ where
 impl<'tcx, T> TransferFunction<'tcx, T>
 where
     Self: Visitor<'tcx>,
-    T: GenKill<Local>,
+    T: TaintDomain<Local>,
 {
     fn t_visit_assign(&mut self, place: &Place, rvalue: &Rvalue) {
         match rvalue {
             // If we assign a constant to a place, the place is clean.
-            Rvalue::Use(Operand::Constant(_)) => self.domain.kill(place.local),
+            Rvalue::Use(Operand::Constant(_)) => {
+                self.state.mark_tainted(place.local);
```

Comment:
Det ville give mere mening for mig at kalde "mark_untainted" her.

Reply:
Ja, det er også det `kill` gør, ved ikke lige hvad der skete der. 😅